strawberry/utils/debug.py

```diff
@@ -1,13 +1,20 @@
 import datetime
 import json
 import typing
+from json import JSONEncoder
+from typing import Any
```

Comment:
This file now uses both `Any` and `typing.Any`. Probably want to make that consistent